package.json

```diff
@@ -6,12 +6,13 @@
   "repository": "https://github.com/suchipi/chai-jest-snapshot",
   "scripts": {
     "build": "rm -rf dist/* && babel src --out-dir dist",
-    "mocha": "mocha -r 'dist/spec/index' 'dist/**/*.spec.js'",
+    "jest": "jest --roots dist/spec-jest",
```

Comment:
We don't need this `jest` script here because it's handled by the example, so let's remove it.